app/src/main/res/values/strings.xml

```diff
@@ -102,11 +102,13 @@
   <string name="order_by_distance">Sort by distance</string>
   <string name="gcm_event_ticker">A new event at your home GDG has been scheduled.</string>
   <string name="ioextended_description">Google I/O Extended is about joining the Google conference at your local group.</string>
+  <string name="gcpnext_description">GCP NEXT Extended is about joining the Google conference at your local group.</string>
```

Comment:
Can you change the description of the event to mention about the conference. Everyone knows Google IO but not much people know gcp